### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kysrpex**
* Add more details to the introduction, add links.
* Modify `resources.yaml` section, add links.
* Remove LICENSE section (that is what GitHub's sidebar is for).

Hopefully the links make it easier for newcomers.

### CODE REVIEW

**Review by sanjaysrikakulam — APPROVED** [review]
Thank you!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
-# VGCN Infrastructure Management
+# VGCN infrastructure management
 
-This repository allows for definition and management of VGNC resources in the
-bwCloud for usegalaxy.eu
+This repository defines and manages the
+[VGNC](https://github.com/usegalaxy-eu/vgcn) compute nodes in the
+[bwCloud](https://www.bw-cloud.org/) for [usegalaxy.eu](usegalaxy.eu)
```
Comment (bgruening) [inline README.md:5]: can we call this bwCloud and de.NBI-cloud to make our funders happy?

Comment (kysrpex) [reply README.md:5]: Like this? I did not fully understand sorry.

```markdown
This repository defines and manages the
[Virtual Galaxy Compute Nodes](https://github.com/usegalaxy-eu/vgcn) on
[bwCloud](https://www.bw-cloud.org/)
(part of [de.NBI-cloud](https://www.denbi.de/))
for [usegalaxy.eu](usegalaxy.eu)
```

Comment (bgruening) [reply README.md:5]: This repository defines and manages the
[Virtual Galaxy Compute Nodes](https://github.com/usegalaxy-eu/vgcn) on
[bwCloud](https://www.bw-cloud.org/) / [de.NBI-cloud](https://www.denbi.de/)
for [usegalaxy.eu](usegalaxy.eu)

**Pull request merged**